src/eradius_log.erl

```diff
@@ -122,12 +134,25 @@ format_cmd(discreq)   -> <<"Disconnect-Request">>;
 format_cmd(discack)   -> <<"Disconnect-Ack">>;
 format_cmd(discnak)   -> <<"Disconnect-Nak">>.
 
+format_cmd_string(request)   -> "Access-Request";
```

Comment:
duplication is going to be removed in later releases, for now i wanted to avoid unnecessary conversions from binaries